rpc_jobs/rpc_aio.yml

```diff
@@ -146,6 +159,9 @@
             RPC_BRANCH using the env vars supplied by ghprb. Ztrigger is a
             jjb variable so uses single braces.
           */
+          if (env.STAGES.contains("Clear Workspace")){{
```

Comment:
The idea was to allow rerunning some stages and re-using state in the workspace. I know that isn't guaranteed to work though as a different workspace may be allocated :/